tests/test_user/test_create_user.py

```diff
@@ -36,3 +40,15 @@ def test_user_duplication(self):
 
         expected_response = user_duplication_mutation_response
         self.assertEqual(query_response, expected_response)
+
+    def test_user_creation_with_non_andela_email(self):
+        """
+        Testing for adding a user with non Andela Email
+        """
+        self.client.execute(non_Andela_email_mutation,
```

Comment:
This seems to be the same code as that for line 51. Kindly share the reason for this.

Reply:
It is unnecessary to attempt creating the same user twice, which is what the code attempts to do.  I have fixed this. @RonKbS thanx for pointing it out.